src/api/Visualization/types.ts

```diff
@@ -398,3 +398,30 @@ export type IUpdateWidgetParams = ICreateWidgetParams;
 export interface IDeleteWidgetParams {
   widgetKey: string;
 }
+
+export interface VisualizationCommentUser extends DeletableUser {
+  hasAccess: boolean;
+}
+
+export interface VisualizationComment {
+  id: number;
+  content: string;
+  user: VisualizationCommentUser;
+  createdAt: string;
+}
+
+export interface CreateVisualizationCommentParams extends IDataSourceParams {
+  visualizationId: number;
+  content: string;
+}
+
+export interface GetVisualizationCommentsParams extends IDataSourceParams {
```

Comment:
You can extend `PaginationClause`, so that  `offset` / `limit` don't have to be repeated:

```typescript
export interface GetVisualizationCommentsParams extends IDataSourceParams, PaginationClause {
  visualizationId: number;
}
```